src/modules/report/base.py

```diff
@@ -1,5 +1,7 @@
 from .collector import services, vulnerabilities, hunters, services_lock, vulnerabilities_lock
 from src.core.types import Discovery
+from __main__ import config
```

Comment:
Note that once [this PR](https://github.com/aquasecurity/kube-hunter/pull/264) is merged, we will stop using `__main__` imports and fetch configuration from separate module.